docs/developer/PROW_CI.md

```diff
@@ -64,12 +64,278 @@ The jobs run can be seen in PRs and in the links in the README.md file of OADP r
 
 OADP operator master branch is tested against the last 3 minor OCP releases. To Update an OCP version the project is tested against, see [Update OCP version](#update-ocp-version).
 
-### TODO Creating new release branch
+### Creating new release branches
 
-https://github.com/openshift/oadp-operator/issues/1227
+To create new OADP release branches (they must follow the pattern `oadp-major.minor`, [example](https://github.com/openshift/oadp-operator/tree/oadp-1.4)):
+- First, create CI files for each new repository branch.
+- Create the release branches in each one of related repositories of OADP, and OADP operator repository itself.
+- The new release branches must be updated:
+  - Downstream repositories release branches may not need updates (except OADP operator, which always need updates).
+  - Upstream repositories release branches need to be rebased.
+
+> **Note**: Try to always create release branches from default branches (usually called master).
+
+> **Note**: Documentation should live only in default branches. For example, for OADP operator, `docs/` and `blogs/` folders can be deleted in release branches.
 
 **TODO automate this process**
 
+#### Example: create CI files for OADP operator repository
+
+CI files for OADP operator (and all of its related repositories) live in https://github.com/openshift/release repository.
+
+For this example, lets say new release branch is `oadp-1.4`.
+
+The new release branch must be added to `core-services/prow/02_config/openshift/oadp-operator/_prowconfig.yaml` changing the following.
+```diff
+...
+   - includedBranches:
+     - master
+     - oadp-1.0
+     - oadp-1.1
+     - oadp-1.2
+     - oadp-1.3
++    - oadp-1.4
+     labels:
+...
+```
+
+Create `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4.yaml`. To make it easier, copy the contents of `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml` (if release branch was created from master branch), changing the following.
+```diff
+...
+ images:
+ - dockerfile_path: Dockerfile
+   from: src
+-  to: oadp-operator
++  to: oadp-operator-1.4
+ promotion:
+...
+ zz_generated_metadata:
+-  branch: master
++  branch: oadp-1.4
+   org: openshift
+   repo: oadp-operator
+```
+
+> **Note**: to get diff between files, you can run `diff -ruN ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4.yaml`.
+
+Create `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.VERSION.yaml` files. To make it easier, copy the contents of `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.VERSION.yaml` files (if release branch was created from master branch), changing the following.
```

Comment:
There is no such file in the release folder:
```shell
$ ls ci-operator/config/openshift/oadp-operator/*VERSION*
ls: cannot access 'ci-operator/config/openshift/oadp-operator/*VERSION*': No such file or directory
```

Reply:
Here is list of files

```shell
$ ls ci-operator/config/openshift/oadp-operator/*
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.14.yaml    ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0.yaml        ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.14.yaml
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.15.yaml    ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.12.yaml  ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.15.yaml
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.16.yaml    ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.13.yaml  ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.16.yaml
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml          ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.14.yaml  ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4.yaml
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0__4.12.yaml  ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.15.yaml  ci-operator/config/openshift/oadp-operator/OWNERS
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.0__4.14.yaml  ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3.yaml

```

Reply:
VERSION is the minor version of OCP, the regex should be `ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-<branch-name>__4.*.yaml`

Reply:
Ah ok so it's not clear then, it needs to be:

```shell
ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.${MINOR_VERSION}.yaml
```